### ISSUE TITLE
Not good validation accuracy in Early training stages

### ISSUE THREAD

**Issue by rog93** [generalized-iou/Detectron.pytorch#4]
I implement GIou loss in Caffe and start training FPN on coco, but the validation accuracy in early stages is much worse than smoothl1 loss. （0.076 vs 0.149 after epoch1, coco-style evaluation metric）
Is that normal?

**Comment by jgwak** [comment]
Sorry I cannot comment anything about something I didn't implement. And I don't know the intermediate validation metrics, since we do not evaluate validation while training. You may try running the code in this repository and evaluating validation accuracy in early stages yourself for comparison with your code.

Closing question. If you believe this issue is relevant to our implementation, please feel free to reopen this issue with concrete reproducible details.
